core/src/main/java/org/elasticsearch/common/geo/builders/ShapeBuilder.java

```diff
@@ -77,16 +77,10 @@
     /** @see com.spatial4j.core.shape.jts.JtsGeometry#index() */
     protected final boolean autoIndexJtsGeometry = true;//may want to turn off once SpatialStrategy impls do it.
 
-    protected Orientation orientation = Orientation.RIGHT;
-
     protected ShapeBuilder() {
```

Comment:
ShapeBuilder seems to hold static references to a `JtsSpatialContext` and a `GeometryFactory`, shouldn't those already throw some kind of error if the dependencies are missing?